docs/developer/contributing.rst

```diff
@@ -223,9 +247,9 @@ To make a branch, open cmder in the pymedphys directory and run the following:
 
 Once you've run that you are now free to make some changes.
 
-Right click on the top level pymedphys directory, and press
-"Open with Code". This document that you're reading is located at
-`docs/developer/contributing.rst`. Use VS Code to navigate to that file and
+Right click on the top level pymedphys directory, and press "Open with Code".
+This document that you're reading is located at
+``docs/developer/contributing.rst``. Use VS Code to navigate to that file and
```

Comment:
I just made these fixes while we're at it. See https://github.com/pymedphys/pymedphys/issues/162